script/deploy/_targetState.json

```
@@ -18,13 +18,16 @@
         "Executor": "2.1.0",
         "FeeCollector": "1.0.1",
         "LiFiDEXAggregator": "1.11.0",
```

Biome (2.1.2)

[error] The key LiFiDEXAggregator was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)